backend/src/core/models/games.aggregate.js

```diff
@@ -0,0 +1,50 @@
+import { Game } from "./game.js";
+
+export class GamesAggregate {
+  games;
+
+  static manyFromDbEntries(games) {
+    const gamesAggregate = new GamesAggregate();
+    gamesAggregate.games = games.map((game) => Game.fromDbEntry(game));
+
+    return gamesAggregate;
+  }
+
+  getIds() {
+    return this.games.map((game) => game.id);
+  }
+
+  isEmpty() {
+    if (this.games.length > 0) return false;
+
+    return true;
+  }
+
+  updateGamesMissingDetails(htmlDetailsPages) {
+    this.games = this.games.map((game) => {
+      const gameCopy = game.copy();
+
+      const page = this.#findGamesHtmlDetailsPage(htmlDetailsPages, gameCopy);
```

Comment:
Very good extraction into a small private method! :+1: 

pass only the game id